README.md

```diff
@@ -1,5 +1,113 @@
+
+![SCE-logo](readme/headImage.png)
+
+# MOVIE BROWSER
+
+
+**LIVE LINK**: [https://tsypko.github.io/movie-browser/](https://tsypko.github.io/movie-browser/)
+
+Current version 1.0.0 
+
+![Thematic Break](readme/thematicBreak.png)
+
+## DESCRIPTION
+
+Explore movies, TV Shows and poeple in easy way. 
+In this app you will find thousands of titles, cast and crew to discover. 
+You will learn their details and viewers' ratings and the data is presented in a friendly way so that you can freely navigate the ocean of movies database.
```

Comment:
Coś mi nie pasuje w tym ostatnim zdaniu. Może lepiej rozbić go na dwa i dać coś w stylu: "You can learn about their details, viewers' ratings and search for some interesting positions. The data is presented in a friendly way so that you can freely navigate the ocean of movies database."

Reply:
Tak, tak będzie lepiej.

Reply:
Mi tu w ogóle jakoś te learn nie pasuje. Może bardziej You can find out about... ;)

Reply:
Mi w sumie też, ale ja od kilku dni na wstecznym chodzę i nie wymyśliłem nic innego. Dzięki za sugestię, zaraz zmieniam.